packages/amplication-client/src/Plugins/PluginsCatalog.tsx

```diff
@@ -30,7 +42,15 @@ const PluginsCatalog: React.FC<Props> = ({ match }: Props) => {
   const handleInstall = useCallback(
     (plugin: Plugin, pluginVersion: PluginVersion) => {
       const { name, pluginId, npm } = plugin;
-      const { version, settings } = pluginVersion;
+      const { version, settings, configurations } = pluginVersion;
+      const requireAuthenticationEntity = configurations
+        ? configurations["requireAuthenticationEntity"]
```

Comment:
Let's change all places the use hard coded "requireAuthenticationEntity" to a const string e.g. const REQUIRE_AUTH_ENTITY = "requireAuthenticationEntity"